### ISSUE TITLE
Remove Account Function

### ISSUE THREAD

**Issue by MartinX3** [vfeskov/gitpunch#60]
I miss a function to remove my account on your service.
Since GitHub integrates this function.

PS:
Thank you for your hard work. :-)

**Comment by vfeskov** [comment]
Yeah I there's no automatic way right now, but you can always drop me a message on 
v[at]vladimirfeskov.com 
and i will delete it for you. Yours has been deleted already.

I will close the issue when i add the function

**Comment by MartinX3** [comment]
Thank you very much! :)
